en/patient-organizations-USA/index.md

```diff
@@ -1,57 +1,53 @@
-# Организации пациентов
+Patient organizations
+For a long time the best patients could do has been organizing some sort of the AA meetings (for the record the first AA fellowship  was  founded in 1937 in the USA and its purpose was to help its members share their experience and feelings). Yet all the help these meetings could provide was emotional support.
 
-Долгое время максимум, на что были способны пациенты, так это организовывать что-то вроде клубов анонимных алкоголиков, первый из которых, кстати, появился в 1937 году в США, чтобы делиться своим опытом и переживаниями. Однако, кроме эмоциональной поддержки, больше никакой пользы эти “клубы” им не приносили.  
-
-Одной из первых громко заявивших о своих правах была Роза Кушнер, которой повезло быть писательницей и не повезло иметь рак груди. В 1970 году в своей очередной книге она подробно описала последствия после проведения радикальной мастэктомии (полного удаления молочной железы), чем вызвала волну обсуждений в обществе. Влияние Розы стало настолько велико, что в итоге ее пригласили в качестве эксперта оценивать релевантность клинических исследований для Национального Института Рака США.
+Rose Kushner was a journalist and pioneering advocate for breast cancer. She was fortunate to have a gift of wrighting but unlucky to have cancer. In 1975 in her book she described in detail the effects of radical mastectomy (the removal of the whole breast) which drew attention of the society. Rose's influence was so significant that she was invited as an expert to assess the relevance of clinical trials for the National Cancer Institute in the USA.
```

Comment:
are you sure the surname is spelled like that? (test comment, don't take it serious)